web/services/AI/ai_model.py

```
@@ -0,0 +1,177 @@
+    # Construct dynamic user input based on example_form
+    user_input = f"""
+    instructions: {AI_prompt_ai_quiz_corrector}
+    quiz_data: {quiz_data}
+    """
+
+    # Generate response
+    try:
+        response = model.generate_content(user_input)
+        return response.text[7:-4]
+
+    except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)